bitnami/external-dns/Chart.yaml

```diff
@@ -24,4 +24,4 @@ sources:
   - https://github.com/kubernetes-sigs/external-dns
   - https://github.com/bitnami/bitnami-docker-external-dns
   - https://github.com/kubernetes-sigs/external-dns
-version: 6.5.6
```

Comment:
I think this is a minor, as you will be updating the name of resources.

Reply:
@joancafom I've updated this but I'm not sure if I did it correctly now. Can you please check?

Reply:
Yes, it is correct now! Thanks 🚀